draft-ietf-tls-esni.md

```diff
@@ -118,12 +118,37 @@ notation comes from {{RFC8446, Section 3}}.
 
 # Overview
 
+ECH is a protocol extension to TLS1.3 which objective is to encrypt the
+ClientHello to fulfill a number of goals (see {{goals}}), in particular
+hiding the destination and therefore the Server Name Indication (SNI).
+
+In order to fulfill these goals ECH introduced and relies on a number
+of key design ideas:
+
```

Comment:
What would the reader learn from this section that cannot be gleaned from the "Table of Contents" ?

Reply:
Well, NO :-) The table of content is just a table of content. The problem is that NO-WHERE there is the that this extension is much more than an extension and that highlights what are the key BIG things that were needed to be designed and the dependencies to make ECH work. And to me neither the Table of Content, nor the introduction, nor the old overview are giving justice to all the work or ECH and to highight the key big inventions, design principles and dependencies. 

We might even want to add some points there.